.travis.yml

```diff
@@ -8,17 +8,14 @@ php:
 matrix:
   fast_finish: true
 
-install:
-  - composer require --dev phpstan/phpstan:^0.12
-
 before_script:
   - composer install
 
 script:
-  - ./bin/php-cs-fixer fix lib/ --dry-run --diff
-  - php ./bin/phpstan.phar analyse -c phpstan.neon lib tests
-  - ./bin/phpunit --configuration tests/phpunit.xml --coverage-clover=coverage.xml
-  
+  - php vendor/bin/php-cs-fixer fix -v --dry-run --stop-on-violation
```

Comment:
We shouldnt stop on violations... having a full report is more helpfull IMO

Reply:
I thought this had to be there to make the travis build fail when violations are found. Will fix.